src/ios-deploy/MobileDevice.h

```diff
@@ -454,6 +454,7 @@ void AMDAddLogFileDescriptor(int fd);
 
 typedef int (*am_device_install_application_callback)(CFDictionaryRef, int);
 
+mach_error_t AMDeviceInstallApplicationBundle(am_device device, CFStringRef path, CFDictionaryRef options, am_device_install_application_callback callback, void *user);
```

Comment:
AMDeviceInstallApplicationBundle doesn't seem to be used anywhere.

Reply:
Ah yes this has been removed.